apps/beeswax/src/beeswax/server/dbms_tests.py

```diff
@@ -46,22 +46,30 @@ def test_get_default(self):
 
     with patch('beeswax.conf.HIVE_SERVER_HOST.get') as HIVE_SERVER_HOST:
       with patch('beeswax.conf.HIVE_SERVER_PORT.get') as HIVE_SERVER_PORT:
-        HIVE_SERVER_HOST.return_value = 'hive.gethue.com'
-        HIVE_SERVER_PORT.return_value=10002
+        with patch('beeswax.server.dbms.KazooClient') as KazooClient:
+          HIVE_SERVER_HOST.return_value = 'hive-llap-1.gethue.com'
+          HIVE_SERVER_PORT.return_value = 10002
```

Comment:
Same as above, we are breaking regular usage.. 

We should not need to update the standard test, but very probably need to add a LLAP Discovery test?